### PR TITLE
Added unit tests for BaseEnum functions

### LINKED ISSUE #67: Unit Tests for BaseEnum
Write Unit Test for Base Enum

### PR DESCRIPTION
**Pull request by laxitshahi**
Resolves #67

### CODE REVIEW

**Review by MohamedAhmed74 — APPROVED** [review]

File: llmproxy/tests/unit_tests/utils/test_enums.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    # Act
+    names = TestEnum.list_names()
+
+    # Assert
+    assert set(names) == test_values
+
+
+def test_list_enum_with_float_values() -> None:
+    # Arrange
+    class TestEnum(float, BaseEnum):
```
Comment (TheVicBro) [inline llmproxy/tests/unit_tests/utils/test_enums.py:50]: Add more test cases, negative numbers, 1.00, etc. if needed

File: llmproxy/tests/unit_tests/utils/test_enums.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    # Act
+    values = TestEnum.list_values()
+
+    # Assert
+    assert set(values) == test_values
+
+
+def test_list_name_with_int_values() -> None:
+    # Arrange
+    class TestEnum(int, BaseEnum):
```
Comment (TheVicBro) [inline llmproxy/tests/unit_tests/utils/test_enums.py:22]: Test for non-existent day (ex. "FOODAY"), if needed.

**Review by TheVicBro — APPROVED** [review]
